index.html

```diff
@@ -7,33 +7,33 @@ <h2 class="header">A galaxy lies open for you to explore.</h2>
 <p>Endless Sky is a 2D space trading and combat game similar to the classic Escape Velocity series. Explore other star systems. Earn money by trading, carrying passengers, or completing missions. Use your earnings to buy a better ship or to upgrade the weapons and engines on your current one. Blow up pirates. Take sides in a civil war. Or leave human space behind and hope to find friendly aliens whose culture is more civilized than your own.</p>
 <p><a href="https://github.com/endless-sky/endless-sky/releases/latest">Downloads</a> are available for Windows, Mac OS, and Linux. Endless Sky is free and open source, the product of a growing community of developers and content creators.</p>
 
-<img class="intro-image" src="/images/introduction/starting-ships.jpg" title="The three ships you can choose between at the start of the game." />
+<img class="intro-image" src="/images/introduction/starting-ships.jpg" title="The three ships you can choose between at the start of the game.">
```

Comment:
Should these have alt attributes too?

Reply:
I'm not sure. They already have titles, which isn't 100% a replacement for alts, but I'm not sure how much extra information I could fit into them.